crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -985,9 +985,10 @@ impl Display for SuiCommand {
                 write!(f, ")")
             }
             Self::Publish(_bytes) => write!(f, "Publish(_)"),
-            Self::Upgrade(_bytes, deps, ticket) => {
+            Self::Upgrade(_bytes, deps, current_package_object_id, ticket) => {
                 write!(f, "Upgrade({ticket},")?;
                 write_sep(f, deps, ",")?;
+                write!(f, ", {current_package_object_id}")?;
```

Comment:
nit, for consistency?

```suggestion
            Self::Upgrade(_bytes, deps, current_package_id, ticket) => {
                write!(f, "Upgrade({ticket},")?;
                write_sep(f, deps, ",")?;
                write!(f, ", {current_package_id}")?;
```